test/functional/test-url-replacements.js

```diff
@@ -200,4 +200,25 @@ describe('UrlReplacements', () => {
     return expect(replacements.expand('?a=P1&b=P2&c=P3&d=OTHER'))
         .to.eventually.equal('?a=abc%20&b=xyz&c=123&d=foo');
   });
+
+  it('should override an existing binding', () => {
+    return expand('ord=RANDOM?', false, {'RANDOM': 'abc'}).then(res => {
+      expect(res).to.match(/ord=abc\?$/);
+    });
+  });
+
+  it('should add an additional binding', () => {
```

Comment:
do we need to update the header/title of the PR too? (just when we built the CHANGELOG)